pkg/cloudprovider/providers/openstack/openstack_volumes.go

```diff
@@ -53,8 +53,12 @@ func (os *OpenStack) AttachDisk(instanceID string, diskName string) (string, err
 			glog.V(4).Infof("Disk: %q is already attached to compute: %q", diskName, instanceID)
 			return disk.ID, nil
 		} else {
-			errMsg := fmt.Sprintf("Disk %q is attached to a different compute: %q, should be detached before proceeding", diskName, disk.Attachments[0]["server_id"])
+			errMsg := fmt.Sprintf("Disk %q is attached to a different compute: %q, detaching", diskName, disk.Attachments[0]["server_id"])
 			glog.Errorf(errMsg)
```

Comment:
Should we be dropping the level of the log message here? Its not an error (anymore anyway) - its one of a few expected conditions.

According to https://github.com/kubernetes/kubernetes/blob/master/docs/devel/logging.md `glog.V(2).Info(errMsg)` would be a better level.